src/src/TemplateBundle/Controller/AssetController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace TemplateBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+
+class AssetController extends CustomTemplateController
+{
+    /**
+     * @Route("/asset/{templateName}/css/{assetName}", name="get_css", defaults={"assetName": "styles"})
+     * @Method("GET")
+     */
+    public function getCssAction($templateName, $assetName, Request $request)
+    {
+        if (empty($assetName)) {
```

Comment:
checeme to tu mat a potom to pouzivat, bez toho aby sme to uvadzali meno stylu? Podla mna to bude matuce.

Reply:
Dal som tam defaulty pre istotu, ale mozeme sa toho kludne zbavit.

Reply:
ja by som sa toho zbavil